### PR TITLE
feat: add empty state to calendar

### PR DESCRIPTION
**Pull request by carissa-tang**
## Implementation description. How did you make this change?
* Added an “empty state” card for days without shifts (on public calendar)
     * Design confirmed with Anya and Stacy
* Update image for reset password email verification page

image

## Steps to test
1. Sign in as donor
2. Schedule shifts and check that the public calendar accurately displays shifts
     * None on Apr 23 - empty state
     * One shift on Apr 23 - one card
     * Another shift on Apr 23 - two cards

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages follow conventional commits and are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s) 
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [ ] The appropriate tests if necessary have been written

### CODE REVIEW

File: frontend/src/components/common/Calendar/WeeklyCalendar.tsx [hunk 1]
```diff
@@ -78,29 +78,71 @@ const DayButton = ({ day }: DayButtonProps) => {
...
+    ) {
+      return false;
+    }
+
+    return true;
+  });
+
+  return !shiftsArr.length
+    ? renderItem({ emptyState: true })
+    : shiftsArr.map((item, index) => {
```
Comment (hanlinc27) [inline frontend/src/components/common/Calendar/WeeklyCalendar.tsx:128]: I'm seeing this crash consistently on this branch: 
image

image

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/Footer.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { Email, Facebook, Instagram } from "./icons";
 
 const Footer = (): JSX.Element => (
   <>
-    <Divider pt={{ base: "0px", md: "79px" }} />
+    <Divider pt={{ base: "79px", md: "117px" }} />
```
Comment (carissa-tang) [inline frontend/src/components/common/Footer.tsx:8]: revert change in my previous PR that made the spacing of all other pages off

**Review by jessiiepeng — APPROVED** [review]
the empty states work on my side for desktop / mobile!

**Review by Shehryar21 — APPROVED** [review]
Checked on staging and was working

**Review by xsharonhe — APPROVED** [review]
worked for me just made a small wording change on main calendar page

**Review by hanlinc27 — APPROVED** [review]

**Pull request merged**